tests/config/cluster_install/ols_configmap.yaml

```diff
@@ -41,3 +41,4 @@ data:
     dev_config:
       enable_dev_ui: true
       disable_auth: false
+      disable_tls: true
```

Comment:
why disable tls?  we are already setting it up today:
https://github.com/openshift/lightspeed-service/blob/f7f3f29415053ad696d77c7bb8bbeadb8554e7f4/tests/config/cluster_install/ols_manifests.yaml#L49-L52

so you just need to add the paths to the tls cert/key files to the config.  The same ones we were passing as args previously.

Reply:
also you need to fix the deployment in ols_manifests.yaml since it's not invoking runner.py today and it's passing tls values as arguments.